api/src/wfl/service/firecloud.clj

```diff
@@ -44,11 +44,25 @@
                                           body-override)
                          :escape-slash false)}
          http/request
-         util/response-body-json
-         :submissionId)))
+         util/response-body-json)))
```

Comment:
I know you probably made this and similar changes to get things to work, but I like this approach.  Let the invocation decide if it wants to toss the rest of the response.